events-api/src/shared/constants/prisma-error-mapping.constant.ts

```diff
@@ -0,0 +1,6 @@
+import { HttpStatus } from '@nestjs/common';
+
+export const ErrorMappings = {
+  P2002: { status: HttpStatus.BAD_REQUEST, message: 'Record with this value already exists.' },
```

Comment:
Tbh, it's more like 500 error since such cases should be handled before we call DB